src/contracts/SavingCircles.sol

```diff
@@ -203,6 +225,79 @@ contract SavingCircles is ISavingCircles, ReentrancyGuard, OwnableUpgradeable {
     return _circle.members[_circle.currentIndex];
   }
 
+  /// @inheritdoc ISavingCircles
+  function isAutomatedDepositsEnabled(address _member) external view override returns (bool) {
+    return automatedDepositsEnabled[_member];
+  }
+
+  /// @inheritdoc ISavingCircles
+  function getEligibleAddressesForDeposit()
```

Comment:
this needs to be refactored, we can't justify iterating over every circle twice, we can probably do it with one iteration

Reply:
this is never performed onchain